signac_dashboard/dashboard.py

```diff
@@ -548,7 +589,22 @@ def modify_query(**new_values):
         def page_not_found(error):
             return self._render_error(str(error))
 
+        @dashboard.login_manager.unauthorized_handler
+        def unauthorized_handler():
+            return self._render_error("Access token is required.")
+
+        @dashboard.app.route("/login")
+        def login():
+            provided_token = request.args.get("token")
+            if provided_token == self.config["ACCESS_TOKEN"]:
+                user = User(provided_token)
+                flask_login.login_user(user)
+                return redirect("/")
```

Comment:
Should we allow the user to provide the token when visiting any route?
(Is that possible?)

Reply:
Yes, it is possible. No, I don't think we should allow it.